src/AISmart.Application.Grains/GAgentBase.cs

```diff
@@ -424,7 +428,21 @@ private async Task HandleEventWithResponseAsync(MethodInfo method, object eventT
         else
         {
             throw new InvalidOperationException(
-                $"The event handler of {eventType.GetType()} needs to have a return value.");
+                $"The Cqevent handler of {eventType.GetType()} needs to have a return value.");
+        }
+
+    }
+
+    protected virtual async void OnExtendedStateChangedAsync()
+    {
+    }
+
+    protected sealed override async void OnStateChanged()
```

Comment:
async void is a possible issue, please research on this. Thanks.

reference:
https://stackoverflow.com/questions/12144077/async-await-when-to-return-a-task-vs-void